### PR TITLE
AB#1807 Configure User Management Console to use User Management Service

### PR DESCRIPTION
**Pull request by sharpedavid**
[AB#1807](https://dev.azure.com/cyrovalente/14004b17-200a-4897-9dd5-d44e111f33a1/_workitems/edit/1807) Configure User Management Console to use User Management Service

### CODE REVIEW

File: backend/src/main/resources/application.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ user-management-client:
     manage-users: manage-users
 config:
   vanity-hostname: localhost
+  allowed-origins: "http://local.umc.com:8080"
```
Comment (sharpedavid) [inline backend/src/main/resources/application.yaml:12]: These settings need to be modified locally to make things work. I was using `/etc/hosts`/ but I typical set-up would just be `*`.

File: frontend/env_config/dev/config.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "idps_to_show": "idir,moh_idp",
```
Comment (sharpedavid) [inline frontend/env_config/dev/config.json:2]: Unrelated change. We never reference `idps_to_show` so I removed it from all the config files.

File: frontend/src/views/AdminEventLog.vue [hunk 3]
```diff
@@ -74,7 +74,6 @@
             </template>
 
         </v-data-table>
-        <button @click="getAllEvents">Refresh</button>
```
Comment (sharpedavid) [inline frontend/src/views/AdminEventLog.vue:77]: Unrelated change. Button used during development, accidentally left in.

**Review by ahoplock — APPROVED** [review]
Reviewed and approved. As noted in Teams, I'm not set up to run the UMC locally so I haven't run the code and just reviewed the changes in GitHub.

File: frontend/env_config/dev/config.json (hunk 2 above)
Comment (tschia) [reply frontend/env_config/dev/config.json:2]: We were using it locally for the testcafe user to be able to login I think
